iceberg/iceberg-rest-server/build.gradle.kts

```diff
@@ -65,6 +77,7 @@ dependencies {
   testImplementation(project(":bundles:aliyun-bundle"))
   testImplementation(project(":bundles:aws-bundle"))
   testImplementation(project(":bundles:gcp-bundle", configuration = "shadow"))
+  testImplementation(project(":bundles:azure-bundle"))
```

Comment:
We are not including `iceberg-azure-bundle` in the embedded tests, correct? Perhaps we should add `reactor-netty-http` and `reactor-netty-core` to ensure that the embedded tests can locate these dependencies.